docs/delivering-your-session.md

```diff
@@ -0,0 +1,39 @@
+# Delivering your session
+
+It's time to go live! Deep breath - you've got this!
+
+Couple of last notes to keep in mind
+
+## Getting started
+
+- Begin your stream five minutes before your scheduled start time with the **Starting soon** scene
+  - Make sure your mic is muted :-D
+- Start with introductions
+  - Introduce yourself and any co-hosts
+    - Try to limit the amount of time spent on introductions
+    - People are there to see you in action!!
+  - Introduce your topic and target audience
+    - Highlight your outline
+    - Set pre-requisites for the audience
+    - Consider touching on topics you won't cover
+
+### During the session
+
+- Answer any questions you feel comfortable based on your [questions plan](./preparing-your-session.md#Preparing-for-questions)
+  - Always repeat the question as not everyone may have seen it
+  - Say the name of the person who asked the question
+- Roll with the punches; mistakes happen
+  - Demos may fail
+  - You may forget something or where you are
+- Bring the energy
+  - Speaking into the void is a little weird
+  - You provide all of the energy
+
+### Closing
+
+- Thank ever
+- Highlight the [Reactor YouTube channel](https://www.youtube.com/channel/UCkm6luGCS3hD25jcEhvRMIA)
+- Highlight any additional resources
+- You're welcome to plug your social media handles
+- Display the closing slide for one minute
```

Comment:
Could you add something here about showing the link to the survey and event code on the closing slide?